pixeldata/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "dicom-pixeldata"
+version = "0.3.0"
+authors = ["Eduardo Pinho <enet4mikeenet@gmail.com>", "Peter Evers <pevers90@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+dicom-object = { path = "../object", version = "0.3.0" }
+dicom-core = { path = "../core", version = "0.3.0" }
+dicom-encoding = { path = "../encoding", version = "0.3.0" }
+dicom-transfer-syntax-registry = { path = "../transfer-syntax-registry", version = "0.3.0" }
+snafu = "0.6.8"
+byteorder = "1.4.3"
```

Comment:
I couldn't use that older version that is used in the rest of dicom-rs because it doesn't have `read_i16_into`.

Reply:
Ah, cool. I didn't realize that. I thought they were the same.

I'm using the `byteorder` `BigEndian/LIttleEndian::read_u/i16_into`, to read in a complete stream of `Vec<u8>` into the correct type. The type is derived from the TransferSyntax. Can this be done with `byteordered` ?

Reply:
If we add that to `byteordered` it means we can replace these long match statements with just 1 line: https://github.com/Enet4/dicom-rs/pull/112/files#diff-bccaaeeafb2e2a338072bbffc91a719da64711a1a879009e771cec2dbcef5148R128